src/services/github.ts

```diff
@@ -0,0 +1,112 @@
+import { Octokit } from '@octokit/rest';
+import { ILogger } from './logger';
+import { IConfig } from 'config';
+import { GithubCommentError } from '../errors/errors';
+import { Payload } from '../entities/job';
+
+export interface IGithubCommenter {
+  getParentPRs(payload: Payload): Promise<Array<number>>;
+  postComment(payload: Payload, pr: number, message: string): Promise<201 | undefined>;
+  updateComment(payload: Payload, pr: number, message: string): Promise<200 | undefined>;
+  getPullRequestCommentId(Payload: Payload, pr: number): Promise<number | undefined>;
+}
+
+export class GithubCommenter implements IGithubCommenter {
+  private _logger: ILogger;
+  private _config: IConfig;
+  private _octokit: Octokit;
+
+  constructor(logger: ILogger, config: IConfig, githubToken: string) {
+    this._logger = logger;
+    this._config = config;
```

Comment:
Looks like we don't use `_config` anywhere after assigning. Do we still need this field?

Reply:
Yep, I was originally unsure as I pared everything down, but it seems unnecessary